scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will run the pre-push checks in the given order
+# Ktlint
```

Comment:
Is this meant to be a list of items? If so, prefer making this a markdown list, e.g.

```bash
# This script will run the pre-push checks in the following order:
#  - ktlint
#  - (others in the future)
```

Reply:
done